docs/tutorials/workflows/ace_flow/ace_flow.rst

```diff
@@ -41,19 +41,23 @@ These models will be included by default in a future release once ACE is publish
    
    The DL models must be in a specific directory structure to be used by the ACE workflow.
 
-   Once you have the models, place them in the required directory structure by using the following command:
+   Once you have the models, place them in the required directory structure by using the following commands:
 
    .. code-block::
 
-      cp <PATH TO UNET MODEL FILE> /code/miracl/seg/models/unet/best_metric_model.pth
-      cp <PATH TO UNETR MODEL FILE> /code/miracl/seg/models/unetr/best_metric_model.pth
+      cp <PATH TO UNET MODEL FILE> <WHERE YOU CLONED MIRACL>/miracl/seg/models/unet/best_metric_model.pth
+      cp <PATH TO UNETR MODEL FILE> <WHERE YOU CLONED MIRACL>/miracl/seg/models/unetr/best_metric_model.pth
 
-   To check that the models are in the correct directory structure, run the following command:
+   The path <WHERE YOU CLONED MIRACL> is the path where you cloned the MIRACL repository. This is the
+   location where you ran these :ref:`installation steps <git clone target>`.
+
+   To check that the models are in the correct directory structure, run the following commands:
 
    .. code-block::
 
-      $ ls /code/miracl/seg/models/unet
-      $ ls /code/miracl/seg/models/unetr
+      docker exec -it <CONTAINER_NAME> bash
```

Comment:
Sorry, I should have updated you on my progress re: the installation script. It automatically mounts `/miracl` to `/code/miracl` in the Docker container now so technically we could ask them here to check the path in the repo they cloned instead of using Docker to do that. That way they wouldn't have to know the container name yet but they should know what the repo path is since they used it above to cp the models to the correct folder. So:

```
$ ls <WHERE YOU CLONED MIRACL>/miracl/seg/models/unet/best_metric_model.pth
$ ls <WHERE YOU CLONED MIRACL>/miracl/seg/models/unetr/best_metric_model.pth
```

What do you think?

Reply:
Yep agreed. I will change it.